invenio/legacy/bibedit/__init__.py

```diff
@@ -17,10 +17,12 @@
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
+"""Legacy BibEdit."""
+
 import warnings
 
-from invenio.utils.deprecation import RemovedInInvenio22Warning
+from invenio.utils.deprecation import RemovedInInvenio23Warning
 
-warnings.warn("BibEdit will be removed in 2.2. Please check "
+warnings.warn("Legacy BibEdit will be removed in 2.3. Please check "
               "'invenio.modules.editor' module.",
-              RemovedInInvenio22Warning)
+              RemovedInInvenio23Warning)
```

Comment:
If you talk about only `run_sql()` calls, maybe yes. Otherwise is quite a larger task. Additionally it relies on BibRecord, and MARCXML in general.